python_modules/dagster/dagster/_core/storage/dagster_run.py

```diff
@@ -532,7 +532,7 @@ class RunsFilter(IHaveNew, LegacyNamedTupleMixin):
 
     """
 
-    run_ids: Sequence[str]
+    run_ids: Optional[Sequence[str]]
```

Comment:
this change is going to merge as part of https://github.com/dagster-io/dagster/pull/24597 but i didn't want to lose the stack history of this branch, so i copied the changes over